tensorflow/training/docker/2.11/py3/Dockerfile.ec2.cpu.os_scan_allowlist.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Double allowlists are required as the shift in Dockerfile file extension change. Requires both EC2 and SM allowlists in both CPU and GPU